examples/ASMapNode/Sample/MapHandlerNode.m

```diff
@@ -218,17 +218,30 @@ - (void)toggleLiveMap
   [_liveMapToggleButton setTitle:liveMapStr withFont:[UIFont systemFontOfSize:14] withColor:[UIColor blueColor] forState:ASControlStateHighlighted];
 }
 
-#pragma mark - Helpers
+- (void)updateLocationTextWithMKCoordinateRegion:(MKCoordinateRegion)region
+{
+  _latEditableNode.attributedText = [self attributedStringFromFloat:_mapNode.region.center.latitude];
```

Comment:
Because the method provides the region as an argument, let's drop _mapNode. from this.  That will also remove all method calls since the other ones in this chain are fast struct field accesses.  Not a big deal, just style.